src/normalizeFallback.js

```diff
@@ -0,0 +1,48 @@
+function normalizeFallbackString(fallbackString, originalOptions) {
```

Comment:
Can we move this to `src/utils/normalizeFallback`, in near future we will release new major version `url-loader` with dropping `node@4` and merge this PR. Thanks!

Reply:
It's been moved!

Reply:
:+1: